docs/environment_variables.md

```diff
@@ -6,6 +6,7 @@ The following environment variables are specifically support in CodeOcean and ar
 |- |- |- |
 | `RAILS_ENV` | `development` | Specifies the Rails environment which can be configured using the files in `config/environments` |  
 | `SECRET_KEY_BASE` | ` ` | Specifies a server-side secret for Rails, must be set for production |
+| `DATABASE_URL` | ` ` | Specifies database parameters to use, rather than those specified in `config/database.yml`. An valid URL would be `postgresql://username:password@host:5432/database?pool=5` |  
```

Comment:
"**A** valid URL would be [...]"

Reply:
Fixed (amended previous commit introducing the line).